standard/classes.md

```diff
@@ -4056,7 +4084,7 @@ Based on the presence or absence of get and set accessors, an indexer is classif
 
 An expression body consisting of “`=>`” followed by an expression `E` and a semicolon is exactly equivalent to the block body `{ get { return E; } }`, and can therefore only be used to specify read-only indexers where the result of the get accessor is given by a single expression.
 
-Even though the syntax for accessing an indexer element is the same as that for an array element, an indexer element is not classified as a variable. Thus, it is not possible to pass an indexer element as a `ref` or `out` argument.
+Even though the syntax for accessing an indexer element is the same as that for an array element, an indexer element is not classified as a variable. Thus, it is not possible to pass an indexer element as an `in`, `out`, or `ref` argument.
```

Comment:
>Thus, it is not possible to pass an indexer element as an `in`, `out`, or `ref` argument. [](http://example.com/codeflow?start=150&length=89)

A ref-returning indexer element can be passed by ref.

Reply:
@cston – Yes, it certainly can. I'm going to suggest:

```suggestion
*Note*: Even though the syntax for accessing an indexer element is the same as that for an array element, an indexer element is not classified as a variable. Thus, it is not possible to pass an indexer element as an `in`, `out`, or `ref` argument unless the indexer itself returns a reference (§FIX_UP_probably_9_7).
```

Making it a *Note*, as it really always was.

I suspect this is an issue stemming from how reference returns impact little corners of the Standard…